src/components/panels/ConfigPanel/BooleanConfig.tsx

```diff
@@ -0,0 +1,115 @@
+import { ReactElement, useEffect, useRef, useState } from "react";
+import classnames from "classnames";
+
+import type { ConfigProps } from "./ConfigPanel";
+
+export default function BooleanConfig({
+  config,
+  selectedConfig,
+  setSelectedConfig,
+  setNewValue,
+}: ConfigProps): ReactElement {
+  const [inputFocused, setInputFocused] = useState(false);
+  const [showUseDefault, setShowUseDefault] = useState(
+    config.value !== config.default
+  );
+  const [localValue, setLocalValue] = useState(config.value);
+  const trueRef = useRef<HTMLInputElement>(null);
+  const falseRef = useRef<HTMLInputElement>(null);
+
+  let inputValue = config.default;
+  if (localValue && config.newValue) {
+    inputValue = localValue;
+  } else if (config.default !== config.value) {
+    inputValue = config.value;
+  }
+
+  useEffect(() => {
+    if (selectedConfig?.name === config.name) {
+      setInputFocused(true);
+    } else {
+      setInputFocused(false);
+    }
+  }, [selectedConfig, config]);
+
+  useEffect(() => {
+    if (config.value === config.default) {
+      setShowUseDefault(false);
+    }
+  }, [config]);
+
+  function handleOptionChange(e: any) {
+    const bool = e.target.value === "true" ? true : false;
+    setNewValue(e.target.name, bool);
+    setLocalValue(bool);
+    if (bool !== config.default) {
+      setShowUseDefault(true);
+    } else {
+      setShowUseDefault(false);
+    }
+  }
+
+  function resetToDefault() {
+    if (config.default) {
+      if (trueRef?.current) {
+        trueRef.current.checked = true;
+      }
+    } else {
+      if (falseRef?.current) {
+        falseRef.current.checked = true;
+      }
+    }
+    setShowUseDefault(false);
+  }
+
+  return (
+    // XXX How to tell aria to ignore the click but not the element?
```

Comment:
This is a smell, the click handler should be on the input, not the input container.

Reply:
We need the click handler on the container as well so that the user can click on the container to highlight the field and show the description on the right. This increases the click/tap area but also works really nice for the radio buttons. I was thinking of using a label instead but I didn't feel it right to try and jam a bunch of extraneous block level elements into the label. 

I'm open to alternative approaches....or an aria fix 👍